File: Algoritmos e programação I/11. (21-08) Lista de Exercícios #4/Beecrowd_1065.cpp
/*
Beecrowd: Problema 1065
14/08/2024
Sophia Swiercoswski
*/

#include <iostream>
using namespace std;

int main() {
    // Ler 5 valores inteiros e contar quantos são pares
    int n, cont = 0;
    for(int i = 1; i <= 5; i++) {
        cin >> n;
        if(n % 2 == 0) {
            cont++;
        }
    }

    // Exibir valores pares
    printf("%d valores pares\n", cont);

    return 0;
}

File: Algoritmos e Programação I/Python/Lista de Exercícios #2/Beecrowd_1065.py
'''
Beecrowd: Problema 1065
03/04/2024
Sophia Swiercoswski
'''

cont = 0

for numero in range(5):
    numero = int(input())
    if numero % 2 == 0:
        cont += 1

print (f"{cont} valores pares")